criu/net.c

```diff
@@ -1818,6 +1819,52 @@ static inline int dump_iptables(struct cr_imgset *fds)
 	return 0;
 }
 
+static inline int dump_nftables(struct cr_imgset *fds)
+{
+	int ret = -1;
+	struct cr_img *img;
+	int img_fd;
+	FILE *fp;
+	struct nft_ctx *nft;
+
+	/* we not dump nftables if nft utility isn't present */
+	if (!check_cmd_exists("nft"))
```

Comment:
Looks like we don't need check_cmd_exists in case we use a static library.

Reply:
yeah. You are right. We need to think about it. Which checks we need to do in that place to decide - dump nft or not. Current check with `check_cmd_exists("nft")` not best solution.